### DISCUSSION TITLE
Progress in the AO3 downloader.

### DISCUSSION THREAD

**Discussion by audiogamer22** [nianeyna/ao3downloader#99] [Ideas]
I'm blind and when I run this program it shows up as a blank line, i'm assuming that's the same for sighted people?

I understand that it's working but I think it would be nice to have, downloading BradsStories.html so I at least know something is going on.

If others don't agree that's completely fine, I can live without this, I just thought i'd put it out there.

Oh, another idea, waht about a sound playing once everything is done?

**Comment by nianeyna** [comment]
It should be printing out *some* information about what's going on to the
console. In particular, each time it starts downloading a new page there
should be a message. If you have multiple pages that are downloading
without showing anything in the console (unless you are downloading marked
for later, which works differently) then something is not working correctly
